NAMESPACE

```diff
@@ -1,5 +1,6 @@
 # Generated by roxygen2: do not edit by hand
 
+S3method(naomi::print,naomi_areas)
```

Comment:
This is odd for two reasons:

* does `print` not work as the generic?
* should naomi not implement this?

Reply:
Thanks, think this was me not understanding when you should use `@exportS3method` over `@export`. Have updated this to `@export` now which I think is correct (from https://roxygen2.r-lib.org/articles/namespace.html#s3) I've fixed the signature too to actually match `print`.